requirements.in

```diff
@@ -1,7 +1,7 @@
 #-----------------------------------
 #   FSD Utils
 #-----------------------------------
-https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.18.tar.gz
+git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1450-is-authenticated-flag
```

Comment:
Needs to be a concrete version before merging

Reply:
Thanks, awaiting new version number on utils

Reply:
Updated to 0.0.21